### PR TITLE
Module/examples

### PR DESCRIPTION
**Pull request by AsysGuptaSF**
Screenshot from 2024-05-28 10-30-10
Screenshot from 2024-05-27 14-32-30
Screenshot from 2024-05-27 13-29-15
Screenshot from 2024-05-27 13-26-02
Screenshot from 2024-05-28 10-30-25

### CODE REVIEW

File: terraform/examples/ec2/main.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module "ec2_instance" {
+  source                        = "../../../modules/ec2"
+  instance_type                 = var.instance_type
+  ami                           = var.ami
```
Comment (mayank0202) [inline terraform/examples/ec2/main.tf]: fetch ami from datalookups

File: terraform/examples/rds/main.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
+module "rds_instance" {
+  source                       = "../../../modules/rds"
+  
+  aws_region                   = var.aws_region
```
Comment (mayank0202) [inline terraform/examples/rds/main.tf]: no need to specify aws region in db , regions are defined in aws provider

**Review by mayank0202 — CHANGES_REQUESTED** [review]
Add `provider.tf` file and `.terraform-version` file

File: terraform/examples/ec2/main.tf (hunk 1 above)
Comment (AsysGuptaSF) [reply terraform/examples/ec2/main.tf]: resolved

File: terraform/examples/rds/main.tf (hunk 2 above)
Comment (AsysGuptaSF) [reply terraform/examples/rds/main.tf]: resolved

**Review by mayank0202 — APPROVED** [review]
LGTM

**Pull request merged**